generator.py

```diff
@@ -348,16 +109,16 @@ def generate_new_sample(template, htmlgrammar, cssgrammar, jsgrammar):
         lambda match: add_html_ids(match, htmlctx),
         html
     )
-    generate_html_elements(htmlctx, _N_ADDITIONAL_HTMLVARS)
+    generate_html_elements(htmlctx, 5)
```

Comment:
Not sure why you made this change (as well as 2 other similar changes in this function). It's cleaner to have these as constants rather than hardcoded numbers.